### PR TITLE
Add BottleDrop (#3556)

### PR DESCRIPTION
**Pull request by arch0345**
BottleDrop is a recycling center in Oregon that provides refunds to deposits on cans/bottles. In addition to self-service centers, they also have bag drop centers (bags of cans/bottles rather than individual) and a kiosk to withdrawal funds via voucher at select locations.

Added BottleDrop (individual self-service), BottleDrop Express (bag drop) as amenity=recycling, and BottleDrop Plus (fund withdrawal kiosk) as amenity=payment_terminal

### CODE REVIEW

**Comment by tas50** [comment]
Closes https://github.com/osmlab/name-suggestion-index/issues/3556

**Comment by arch0345** [comment]
Looks like I missed a colon

File: data/brands/amenity/recycling.json [hunk 1]
```diff
@@ -10,6 +10,42 @@
...
+      "displayName": "BottleDrop Express",
+      "id": "bottledropexpress-f8d7ba",
+      "locationSet": {"include": ["us"]},
+      "matchNames": ["bottle drop"],
+      "tags": {
+        "amenity": "recycling",
+        "brand": "BottleDrop",
+        "brand:wikidata": "Q107673127",
+        "name": "BottleDrop Express",
+        "note": "BottleDrop Express is the bag drop service. For self service, use BottleDrop instead.",
```
Comment (tas50) [inline data/brands/amenity/recycling.json:41]: @bhousel is a note how we handle messages to the user in editors?

File: data/brands/amenity/payment_terminal.json [hunk 2]
```diff
@@ -4,6 +4,18 @@
     "exclude": {"generic": ["^payment terminal$"]}
   },
   "items": [
+    {
+      "displayName": "BottleDrop Plus",
```
Comment (tas50) [inline data/brands/amenity/payment_terminal.json:8]: Are these all actually BottleDrop Plus branded. I know there's a kiosk in some Wholefoods and Trader Joes locations, but they're not in the Bottle Drop Plus 20% program.

Comment (arch0345) [reply data/brands/amenity/payment_terminal.json:8]: You're right, some of the kiosks aren't BottleDrop Plus branded. I'll add another entry with just 'BottleDrop'.

File: data/brands/amenity/recycling.json (hunk 1 above)
Comment (bhousel) [reply data/brands/amenity/recycling.json:41]: No the `note` isn't shown anywhere else - it's just a note for the name-suggestion-index team..

**Comment by arch0345** [comment]
Should I move the note outside of tags?

**Comment by bhousel** [comment]
This looks ok to me - can I merge it?

**Comment by arch0345** [comment]
> This looks ok to me - can I merge it?

Yeah, all issues have been resolved

**Pull request merged**